src/main/java/org/jlab/clas/timeline/analysis/alert/alert_atof_tdc.groovy

```diff
@@ -22,19 +24,27 @@ def data = new ConcurrentHashMap()
       if (component <= 10) file_index = String.format('sector%d_layer%d_component%d_order0', sector, layer, component)
       else file_index = String.format('sector%d_layer%d_component%d_order1', sector, layer, component-1)
       def h1 = dir.getObject(String.format('/ALERT/TDC_%s', file_index))
-      data[run].put(String.format('atof_tdc_%s', file_index),  h1)
-      def f1 = ALERTFitter.tdcfitter(h1)
-      data[run].put(String.format('fit_atof_tdc_%s', file_index),  f1)
-      data[run].put(String.format('peak_location_atof_tdc_%s', file_index),  f1.getParameter(1))
-      data[run].put(String.format('sigma_atof_tdc_%s', file_index),  f1.getParameter(2))
-      data[run].put(String.format('integral_normalized_to_trigger_atof_tdc_%s', file_index),  Math.sqrt(2*3.141597f) * f1.getParameter(0) * f1.getParameter(2)/trigger.getBinContent(reference_trigger_bit) )
+      if(h1!=null && h1.getEntries()>10) {
```

Comment:
fyi @Sangbaek, I'm not sure if `10` is a good threshold here, feel free to adjust if you need.